OpenRA.Mods.Common/Traits/ExternalCapturable.cs

```diff
@@ -21,6 +21,7 @@ public class ExternalCapturableInfo : ITraitInfo
 		public readonly bool AllowAllies = false;
 		public readonly bool AllowNeutral = true;
 		public readonly bool AllowEnemies = true;
+		public readonly bool AutoTargetIgnores = false;
```

Comment:
Please add a `desc` (like the one below) to this trait property describing the circumstance that this is tied to.